pyrokinetics/gs2.py

```diff
@@ -397,6 +415,33 @@ def load_numerics(self, pyro, gs2):
         grid_type = gs2['kt_grids_knobs']['grid_option']
         numerics = Numerics()
 
+        # Set no. of fields
+        if 'fphi' in gs2['knobs'].keys():
+            if gs2['knobs']['fphi'] > 0.0:
+                numerics.phi = True
+            else:
+                numerics.phi = False
+        else:
+            numerics.phi = True
+
+        # Set no. of fields
+        if 'fapar' in gs2['knobs'].keys():
+            if gs2['knobs']['fapar'] > 0.0:
+                numerics.apar = True
+            else:
+                numerics.apar = False
+        else:
+            numerics.apar = False
+
+        # Set no. of fields
+        if 'fbpar' in gs2['knobs'].keys():
+            if gs2['knobs']['fbpar'] > 0.0:
+                numerics.bpar = True
+            else:
+                numerics.bpar = False
+        else:
+            numerics.bpar = False
```

Comment:
This can use `dict.get()` again:

```suggestion
        # Set no. of fields
        numerics.phi = gs2["knobs"].get("fphi", 0.0) > 0.0
        numerics.apar = gs2["knobs"].get("fapar", 0.0) > 0.0
        numerics.bpar = gs2["knobs"].get("fbpar", 0.0) > 0.0
```